source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/list/__tests__/invite-user-acceptance-test.tsx

```diff
@@ -206,17 +209,19 @@ describe('Acceptance | User list | Invite User', () => {
           });
 
           describe('clears error on close', () => {
+            let modal;
             beforeEach(async () => {
-              await page.inviteUser();
+              modal = new InviteUserModalInteractor('[data-test-users-invite-user-modal]');
```

Comment:
this was an intermittently failing test, and I had very little trust in the OG interactor, so I `new`d a new one.
also, I think this test may be testing something that can't actually happen, I don't remember at this point. :-\